dist/css/blower.css

```diff
@@ -370,6 +370,24 @@
       --color-alert-text: #0f172a;
       --color-alert-border: #e2e8f0;
     }
+    .alert[class*=alert--text-] {
+      --spacing-alert: 0;
+      --color-alert-background: transparent;
+      --border-width-alert: 0;
+      display: inline-block;
+    }
+    .alert--text-success {
+      --color-alert-text: #3F9A7A;
+    }
+    .alert--text-danger {
+      --color-alert-text: #D75052;
+    }
+    .alert--text-warning {
+      --color-alert-text: #DF8260;
+    }
+    .alert--text-info {
+      --color-alert-text: #00B8D4;
+    }
```

Comment:
Bin ich kein Freund von. Das geht ein bisschen gegen BEM.

Mein Vorschlag wäre, dass man hier `.alert--text` und `.alert--${status}` kombiniert und dann die entsprechenden Custom Properties überschreibt.

Schau dir gerne mal an, wie das bei den Button-Klassen gelöst ist. Hier wird z.B. ein destruktiver Outline-Button wie folgt definiert:

```html
<button class="button button--destructive button--outline">
	Destructive
</button>
```

Die Klassen sind also `button`, `button--outline` und `button--destructive` und nicht `button`, `button--outline-destructive`.

--- 

Ich würde an der Stelle mal zur Debatte stellen, ob wir nicht vom BEM-Standard hier etwas abweichen wollen und stattdessen zu `is-text` und `is-success` wechseln. Dann ist das Konsistent zwischen allen Komponenten. Könnte aber eben auch verwirrend sein, weil `.button.is-text` nicht dieselben Styles anwenden, wie `.alert.is-text`, sondern eben je nach Komponente unterschieden wird. An sich ein Teil von BEMIT.

Das wäre allerdings ein Breaking Change, der nicht in diesem Pull Request bearbeitet werden sollte, sondern in einem separaten Ticket.

Reply:
Ich glaube, ich verstehe, was du meinst, hatte es mit diesem Ansatz versucht, bin allerdings nicht zu dem gewünschten Ergebnis gekommen. Meine Erkenntnis war, es muss viel überschrieben werde und die Farben können leider nicht aus den vorhandenen vars recycelt werden (da schlicht nicht vorhanden). So, hab ich mich dann für diese Variante entschieden. Also in irgendeiner Weise müssen die Text-Varianten ihre eigene Farbe mitbringen, sonst ginge das wahrscheinlich ganz ok. Oder `.alert--${status}` speichert diese bereits, aber das ist auch etwas komisch, oder?